README.md

```diff
@@ -14,27 +14,34 @@ and add to your path.
 
 # Usage 
 
-Before start using the CLI you should initialize it.
-For the initialization you have to provide:
-- initialization file: [corectl-init-example.yaml](corectl-init-example.yaml)
+Before usage, you should initialize. It sets up your github integration with your developer environments.
+It requires the following:
+- initialization file: [init-example.yaml](examples/init-example.yaml)
 - your person GitHub token to perform operations on your behalf. See more info [here](#GitHub-Access-Token)
 
 To run initialization run:
 ```bash
 corectl config init
 ```
-It will save a few configuration options and clone configuration repositories:
-`cplatform-environments` and `software-templates`.
 
-Please note that you should periodically update local `corectl` configuration by running:
+Saves configuration options and clones repositories: `cplatform-environments` and `software-templates`. 
+
+`cplatform-environments` - repository that holds the configuration settings and parameters for core platform environments.
+
+`software-templates` - repository featuring bootstrap templates designed for quick project setups.
+
+The repositories mentioned above are owned by CECG and are available for testing purposes. To gain access, you must be added to the appropriate organization and granted the necessary permissions.
```

Comment:
Are they owned by CECG?

I would expect `cplatform-environments` will be owned by the client and we just help to set it up
And `software-templates` is a public one, but clients might create their own private set of templates